### DISCUSSION TITLE
How can I help translate Transmission?

### DISCUSSION THREAD

**Discussion by hanklank** [transmission/transmission#2494] [Q&A]
I can't see anywhere where it's mentioned that translation nowadays should take place at
https://www.transifex.com/transmissionbt/transmissionbt/dashboard/
also - the projects launchpad page should be updated with this little info, as translating there is misleading. https://launchpad.net/transmission

**Comment by mikedld** [comment]
https://github.com/transmission/transmission/wiki/Translating

Translations on Launchpad are closed. @jordanl assured me that necessary information is present there where translators could see it.

**Comment by hanklank** [comment]
Great!! Just looked again at the launchpad page, and I cant find any information about that there. Just a link to transifex/or the wiki and a one one line sentence would be enough there I think. 

And, thanks for keeping transmission going!!

**Comment by buhtz** [comment]
Could be closed or not?

**Comment by hanklank** [comment]
@buhtz You can still translate in launchpad by choosing any of the links at https://translations.launchpad.net/transmission, please try. (even if the translation isn't used further).
A sentence on the same page linked above says transmission doesn't use launchpad - Maybe a link to https://github.com/transmission/transmission/wiki/Translating there would make it easier to not miss if it's not possible to turn of the translations there.
